### DISCUSSION TITLE
netlify部署一直报错失败

### DISCUSSION THREAD

**Discussion by yueziji** [spencerwooo/onedrive-vercel-index#103] [Q&A]
从vercel/heroku上部署是正常的，但在netlify上部署同样的项目一直失败，不知道什么原因，很好奇
下面是报错
9:37:34 PM: Failed to compile.
9:37:34 PM: 
9:37:34 PM: ModuleNotFoundError: Module not found: Error: Can't resolve 'canvas' in '/opt/build/repo/node_modules/pdfjs-dist/build'
9:37:34 PM: > Build error occurred
9:37:34 PM: Error: > Build failed because of webpack errors
9:37:34 PM:     at /opt/build/repo/node_modules/next/dist/build/index.js:397:19
9:37:34 PM:     at async Span.traceAsyncFn (/opt/build/repo/node_modules/next/dist/telemetry/trace/trace.js:60:20)
9:37:34 PM:     at async Object.build [as default] [(/opt/build/repo/node_modules/next/dist/build/index.js:77:25)

**Comment by spencerwooo** [comment]
Netlify 是之前的一个评论给的解决方案，我自己没尝试过去 Netlify 上部署，所以能用 Vercel 的话还是用 Vercel？

**Reply by yueziji** [reply]
在用vercel/heroku是正常的，但netlify就很神奇的一直失败，也不知道为什么😂

**Reply by spencerwooo** [reply]
用的这个方法？https://github.com/spencerwooo/onedrive-vercel-index/issues/19#issuecomment-892538577

**Reply by yueziji** [reply]
是的，这个方法貌似行不通😂
